front/src/modules/ui/object/record-table/components/RecordTableHeader.tsx

```diff
@@ -210,6 +213,11 @@ export const RecordTableHeader = () => {
                       variant="tertiary"
                       Icon={IconPlus}
                       position="middle"
+                      style={{
```

Comment:
@Kanav-Arora, I'm not a fan of this approach as this is introducing a way of overwriting the style of our basic components which is not something we want for now as we want to enforce consistency and to be sure that we are not breaking components when updating their API ; this style API is too flexible.

I think here we should actually not use an IconButton. Could you instead :
- add the hover on all RecordTableHeader
- add a onClick on the + Cell itself
- remove this IconButton to use the Icon itself direclty?

Reply:
Also we should make sure to check the behavior when the columns are taking the whole page width

Reply:
Okay I'll proceed with it